subprojects/packagefiles/protobuf/meson.build

```diff
@@ -66,13 +66,20 @@ libutf8_range_src = files(
   'third_party/utf8_range/naive.c',
   'third_party/utf8_range/range-avx2.c',
   'third_party/utf8_range/range-neon.c',
-  'third_party/utf8_range/range-sse.c',
   'third_party/utf8_range/utf8_validity.cc',
 )
+libutf8_range_args = []
+if build_machine.cpu_family() == 'x86_64'
+  # range-sse.c is wrapped in #ifdef __x86_64__
+  libutf8_range_src += files('third_party/utf8_range/range-sse.c')
+  # It requires -march=native (or some other more targetted way)
+  # to be able to use SSE.
```

Comment:
I can surely think of a targeted way to be able to use SSE!

Reply:
I tried to do that now. This complicates the logic, because compilers with gcc argument syntax, compilers with msvc argument syntax and strangely enough clang-cl all behave a bit different from each other.

I first wanted to leave `-march=native` as is because I assumed it worked everywhere since it was there before, and I wanted to make the least amount of changes to fix the issue.